advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -13,9 +13,9 @@ This section contains a general overview of topics that you will learn in this l
 
 ### Web Content Accessibility Guidelines
 
-<span id="wcag-purpose">The WCAG exist in order to help create a shared standard when it comes to web accessibility.</span>Think of web accessibility as the destination, and the WCAG as one of the tools that can help get us closer to it.
+<span id="wcag-purpose">The WCAG exist in order to help create a shared standard when it comes to web accessibility.</span> Think of web accessibility as the destination, and the WCAG as one of the tools that can help get us closer to it.
 
-It's important to understand that while the WCAG can be incredibly helpful for building an a11y foundation, they are not the finish line when it comes to accessibility. Like the name says, they are guidelines, and they will only help us make websites *more* accessible.
+It's important to understand that while the WCAG can be incredibly helpful for building an a11y foundation, they are not the finish line when it comes to accessibility. Like the name says, they are guidelines, and they will only help us make websites _more_ accessible.
```

Comment:
For the markdown we use asterisk. Not the _ _. If you could revert it back to the asteriks that would be great! You can check it out in this part of the [style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#markdown-styling) if your curious.